frontend/src/components/CarCard.jsx

```diff
@@ -30,7 +31,7 @@ function CarCard(props) {
           {userVehicle.brand} | {userVehicle.model}
         </h1>
         <h3>
-          {userVehicle.enginePower} ({userVehicle.horsePower} CH)
+          {userVehicle.enginePower} ({userVehicle.horsePower} CH){" "}
```

Comment:
(Does this new `{" "}` serve a purpose, or is it an artefact ? If it is the latter, please remove it)

Reply:
Prettier transforms the space between ")" and next variable with that when I save. But we need that space between the two.